src/assets/source/sass/components/_table.scss

```diff
@@ -1,5 +1,5 @@
-.user-role {
-  margin-right: 10px;
+.container-fluid{
+  width: 50%;
```

Comment:
.container-fluid class is meant to be full width.
If you want your content to be 50% width add it in a container with a custom class and make it 50% width.